src/features/auth/authApi.js

```diff
@@ -0,0 +1,27 @@
+import axiosInstance from '../../axiosConfig';
+
+function registerUser(data) {
+  const { username, email, password } = data;
+
+  return axiosInstance.post('/users', {
+    user: {
+      name: username,
+      email,
+      password,
+    },
+  });
+}
```

Comment:
Hello @Estherstarr 
You have done a great job 🥂 
However, if you look at this part, I think your user data is missing an `id`, I think we should add it, you can use a tool like  nanoid that generates unique `ID`
`user: {
     id: nanoid(),
    ......
}
`
Kindly let me know your thoughts on this, otherwise great job 👍🏿